### ISSUE TITLE
Service Menue C2 Update to webOS23

### ISSUE THREAD

**Issue by LGdet** [Maassoft/ColorControl#285]
There are already firmware updates for the 2022 model series, which upgrade from webOS22 to webOS23.
It would be necessary to add the full service menu mode also for C2/G2 models if possible. Thank you

**Comment by Maassoft** [comment]
So the new WebOS23 for 2022 model series will also lock you out of the full service menu?
I'll add support for the 2022 model numbers in the next release.

**Comment by LGdet** [comment]
Hello, yes it is indeed like that. If you update a C2/G2 of model year 2022 to webOS23, you will have the limited service menu of the C/G3 series afterwards.It's great that you're including this in the next versionGreetings from GermanyDetlefVon meinem iPhone gesendetAm 30/11/2023 um 17:42 schrieb Maassoft ***@***.***>:﻿
So the new WebOS23 for 2022 model series will also lock you out of the full service menu?
I'll add support for the 2022 model numbers in the next release.

**Comment by MarcinTomaszTomaszewski** [comment]
Yes, I can confirm. LG C2 42 service menu not working, at least for the 03.33.90 webos build

**Comment by AndreyArbaiten** [comment]
> So the new WebOS23 for 2022 model series will also lock you out of the full service menu? I'll add support for the 2022 model numbers in the next release.

Good day. Also, after updating to WebOS23, on 2022 models such as OLED48A2 there is no access to the full service menu. Could you please add support for A2.
